### DISCUSSION TITLE
Doesn't seem working...

### DISCUSSION THREAD

**Discussion by maicol07** [cssninjaStudio/unplugin-fonts#5] [General]
Hi,
I'm using [laravel-vite](https://laravel-vite.netlify.app) and it seems your plugin isn't working. This is my current config:
```
ViteFonts({
      google: {
        families: ['Montserrat', 'Nunito']
      }
    })
```
I don't know if there is anything wrong... Can you help me?
Thanks

**Comment by stafyniaksacha** [comment]
Hum, I have not tried laravel-vite.

The documentation says that we have to load plugins like this:

```ts
// vite.config.ts
import { defineConfig } from "laravel-vite";
import ViteFonts from "vite-plugin-fonts";

export default defineConfig()
  .withPlugins(
    ViteFonts({
      google: {
        families: ['Montserrat', 'Nunito']
      }
    }),
    // other plugins ...
 );
```

**Reply by maicol07** [reply]
Not working, unfortunately.
I have another Vite plugin loaded as the standard configuration and works well

**Reply by stafyniaksacha** [reply]
Do you have a link to the other plugin that works ?  

I suspect that `transformIndexHtml` ([used here](https://github.com/stafyniaksacha/vite-plugin-fonts/blob/main/src/index.ts#L15)) from vite is not implemented in `laravel-vite` ([maybe here](https://github.com/innocenzi/laravel-vite/blob/main/src/ManifestEntry.php))

**Reply by maicol07** [reply]
> 
> 
> Do you have a link to the other plugin that works ?

This one: https://github.com/bunchtogether/vite-plugin-flow
I don't know if it is useful, since it's a plugin that works before building JS

**Reply by danimalweb** [reply]
I have a very similar issue when running Vite in a WordPress environment. There is no index.html file to inject into.
